go.mod

```diff
@@ -116,3 +116,4 @@ require (
 )
 
 go 1.14
+replace github.com/ipld/go-ipld-prime => ./../../ipld/go-ipld-prime/
```

Comment:
removal TODO of course

Reply:
replaced with the HEAD commit from https://github.com/ipld/go-ipld-prime/pull/203

which is still temporary